src/assets.ts

```diff
@@ -13,9 +13,9 @@
 //
 // You should have received a copy of the GNU General Public License
 // along with provider.  If not, see <http://www.gnu.org/licenses/>.
-import { Asset, AssetsResolver } from "@prosopo/contract/types";
+import { Asset, AssetsResolver } from "@prosopo/contract";
```

Comment:
this is what should be used @prosopo/contract/types ref. https://github.com/prosopo-io/provider/issues/130